service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/endpoints/RestaurantRoutes.kt

```diff
@@ -51,6 +54,18 @@ fun Route.restaurantRoutes() {
             val result = discoverRestaurant.getRestaurantsByIds(restaurantIds)
             call.respond(HttpStatusCode.Created, result.toDto())
         }
+
+        get("/search/{query}") {
```

Comment:
What will happen when we pass query with empty string? Should have a default list returned?